src/encoded/tests/test_permissions.py

```diff
@@ -228,21 +228,21 @@ def test_users_view_basic_authenticated(submitter, authenticated_testapp):
     res = authenticated_testapp.get(submitter['@id'])
     assert 'title' in res.json
     assert 'email' not in res.json
-    assert 'access_keys' not in res.json
+    assert not res.json.get('access_keys', '')
```

Comment:
This seems like a suboptimal change. `assert 'access_keys' not in res.json` is more idiomatic Python for checking existence of a key in a dict. Also `assert not res.json.get('access_keys', '')` returns `None` by default so doesn't need the default empty string.